src/currencydialog.cpp

```diff
@@ -264,18 +268,18 @@ void mmCurrencyDialog::OnOk(wxCommandEvent& /*event*/)
     else
         m_currency->BASECONVRATE = 1;
     
-    if ((m_currency->GROUP_SEPARATOR == m_currency->DECIMAL_POINT)
-        || (!wxString(" .,").Contains(m_currency->GROUP_SEPARATOR)))
-        return mmErrorDialogs::ToolTip4Object(grpTx_
-            , _("Invalid Entry")
-            , _("Grouping Char"));
-
     if (!wxString(".,").Contains(m_currency->DECIMAL_POINT)
         || m_currency->DECIMAL_POINT.empty())
         return mmErrorDialogs::ToolTip4Object(decTx_
             , _("Invalid Entry")
             , _("Decimal Char"));
 
+    if ((m_currency->GROUP_SEPARATOR == m_currency->DECIMAL_POINT)
```

Comment:
👍 `m_currency->DECIMAL_POINT.empty()` must be checked before to place pop-up in correct place